lib/services/plugins-service.ts

```diff
@@ -45,7 +45,14 @@ export class PluginsService implements IPluginsService {
 			corePlugins = <string[]>_.union(this.$project.getProperty(PluginsService.CORE_PLUGINS_PROPERTY_NAME, this.$projectConstants.DEBUG_CONFIGURATION_NAME), this.$project.getProperty(PluginsService.CORE_PLUGINS_PROPERTY_NAME, this.$projectConstants.RELEASE_CONFIGURATION_NAME));
 		}
 
-		return _.map(corePlugins,(pluginIdentifier: string) => this.identifierToPlugin[pluginIdentifier]);
+		return _.map(corePlugins, pluginIdentifier => {
+			let plugin = this.identifierToPlugin[pluginIdentifier];
+			if (!plugin) {
+				this.$errors.failWithoutHelp("Invalid plugin '%s', check your configurations and remove it manually", pluginIdentifier);
```

Comment:
You have enabled an invalid plugin: %s. Check your `.debug.abproject` and `.release.abproject` files in the project root and correct or remove the invalid plugin entry.